### PR TITLE
feat: add mim legacy and pusd ir vault

### PR DESCRIPTION
**Pull request by hellodespereaux**

### CODE REVIEW

File: api/v2/rateVaults/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import groupBy from 'lodash/groupBy'
+
+import { VAULTS } from '../../../helpers/v2/constants'
+import getIRVaultData from '../../../helpers/v2/getIRVaultData'
+import {utils as ethersUtils} from "ethers/lib/ethers";
```
Comment (witherblock) [inline api/v2/rateVaults/index.js:5]: arrange deps

File: api/v2/rateVaults/index.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import groupBy from 'lodash/groupBy'
```
Comment (witherblock) [inline api/v2/rateVaults/index.js:1]: file name should be irVaults

File: api/v2/rateVaults/index.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                VAULTS.map((vault) => {
+                    return getIRVaultData(vault)
+                })
+            );
+
+        const vaultArray = VAULTS.map((item, index) => {
+            return {
+                ...item,
+                tvl: ethersUtils.formatUnits(data[index].tvl, 18),
+                rate: (data[index].rate.toNumber() / 10 ** 8).toString(),
```
Comment (witherblock) [inline api/v2/rateVaults/index.js:19]: why not use ethers utils here?

File: helpers/v2/constants.js [hunk 4]
```diff
@@ -340,3 +340,36 @@ export const SSOVS = [
...
+        underlyingSymbol: '2CRV',
+        symbol: 'MIM3CRV-1',
+        version: 1,
+        chainId: 42161,
+        collateralDecimals: 18,
+        duration: 'monthly',
+        retired: true,
+        address: '0x602B9f9B7126DEce2b9a589B49Fe5947c903A156',
+    }
+]
```
Comment (witherblock) [inline helpers/v2/constants.js:375]: duration is incorrect

**Review by witherblock — CHANGES_REQUESTED** [review]
yarn.lock is not updated

File: api/v2/rateVaults/index.js (hunk 1 above)
Comment (hellodespereaux) [reply api/v2/rateVaults/index.js:5]: done

File: api/v2/rateVaults/index.js (hunk 2 above)
Comment (hellodespereaux) [reply api/v2/rateVaults/index.js:1]: done

File: api/v2/rateVaults/index.js (hunk 3 above)
Comment (hellodespereaux) [reply api/v2/rateVaults/index.js:19]: done

File: helpers/v2/constants.js (hunk 4 above)
Comment (hellodespereaux) [reply helpers/v2/constants.js:375]: done

**Review by witherblock — APPROVED** [review]

**Pull request merged**